pytripgui/controller/tree_cont_aux.py

```diff
@@ -0,0 +1,26 @@
+class DosCollection(object):
+    type = 'DOSCUBES'
```

Comment:
This is redundant - information about the type is already stored in a class.
Why not create a several class and then set the type upon object creation ?